include/UIKit/UIPasteboard.h

```diff
@@ -20,7 +20,8 @@
 #import <Foundation/NSObject.h>
 
 @class NSString;
-@class NSArray;
+@class NSArray<ObjectType>;
```

Comment:
NSArray<ObjectType>is how you forward-declare a generic type.

---
In reply to: [88955275](https://github.com/Microsoft/WinObjC/pull/1433#discussion_r88955275) [](ancestors = 88955275)